app.config.ts

```diff
@@ -1,10 +1,111 @@
 import { ExpoConfig, ConfigContext } from '@expo/config'
+import * as dotenv from 'dotenv'
+
+dotenv.config()
+
+const environments = {
+  prod: 'production',
+  stg: 'staging',
+  qa: 'qa',
+} as const
+
+export const envValues = Object.values(environments)
+export type Environments = typeof envValues[number]
+
+type Setup = { [key in Environments]: string }
+
+const adaptiveIconPath = './assets/icons/android/adaptive-icon-'
+const appIconPath = './assets/icons/ios/icon-'
+
+// CONFIG: Add your eas build config here !!
+export const APP_CONFIG = {
+  androidPackageName: 'your_android_package_name', // CONFIG: Add your android package name here
+  appName: 'your_app_name', // CONFIG: Add your app name here
+  easProjectId: 'your _eas_project_id', // CONFIG: Add your eas project ID here
+  iosBundleIdentifier: 'your_ios_bundle_identifier', // CONFIG: Add your ios bundle identifier here
+  isDev: process.env.IS_DEV === '1',
+  isExpoGo: process.env.IS_EXPO_GO === '1',
+  schema: 'your_schema_name', // CONFIG: Add your schema name
```

Comment:
Maybe we should add more context what is schema? For other people this might be confusing 🤔

Reply:
I guess we could even remove schema from the config for now. Probably most project will be managed workflow, so it won't have any impact. 


![Zrzut ekranu 2022-10-27 o 13 10 12](https://user-images.githubusercontent.com/60507270/198269212-2b65f3b9-8287-4bf3-a3a3-10d5080bb9e5.png)

I think it will be even better to add link to docs, where every parameter has it description. WDYT @MateuszRostkowski ?

Reply:
Ok, i've removed schema from config, and add link to docs